experiment/references.md

```diff
@@ -1 +1,12 @@
-### Link your references in here
\ No newline at end of file
+
+
+  -  https://www.scribd.com/doc/136565/Mechanics-of-Materials-Torsion-Test
```

Comment:
Please use the agreed upon format for references

Reply:
Only the Simulation Folder part of the code was done. This requires help from SME.